workflow/controller/operator_test.go

```diff
@@ -6843,7 +6843,7 @@ func TestNoPodsWhenShutdown(t *testing.T) {
 
 	node := woc.wf.Status.Nodes.FindByDisplayName("hello-world")
 	if assert.NotNil(t, node) {
-		assert.Equal(t, wfv1.NodeSkipped, node.Phase)
+		assert.Equal(t, wfv1.NodeFailed, node.Phase)
```

Comment:
Double check this. 

Not sure why it was NodeSkipped before

Reply:
If the Pod never ran at all, NodeSkipped could make sense. The source (this is the test) is marking it prior to it running.

Reply:
Okay reverting this

Reply:
#13214 is changing this to `NodeFailed` for a good reason: `NodeSkipped` breaks retries. Otherwise I'm still not sure why it was `NodeSkipped` initially.

See also #11849 for the "Cancelled" status that would more accurately describe this